app/controllers/api/database.php

```diff
@@ -297,38 +299,48 @@
     ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_ATTRIBUTE)
-    ->param('collectionId', null, new UID(), 'Collection unique ID. You can create a new collection using the Database service [server integration](/docs/server/database#createCollection).')
-    ->param('id', null, new Text(256), 'Attribute ID.')
+    ->param('collectionId', '', new UID(), 'Collection unique ID. You can create a new collection using the Database service [server integration](/docs/server/database#createCollection).')
+    ->param('id', '', new Text(256), 'Attribute ID.')
     ->param('type', null, new Text(256), 'Attribute type.')
-    ->param('size', null, new Integer(), 'Attribute size.')
-    ->param('required', null, new Boolean(), 'Is required flag.')
-    ->param('signed', true, new Boolean(), 'Is signed flag.')
-    ->param('array', null, new Boolean(), 'Is array flag.')
+    // TODO@kodumbeats add units to description
+    ->param('size', null, new Numeric(), 'Attribute size.')
+    ->param('required', null, new Boolean(), 'Is attribute required?')
+    ->param('signed', null, new Boolean(), 'Is attribute signed?')
+    ->param('array', null, new Boolean(), 'Is attribute an array?')
+    // TODO@kodumbeats "We should have here a whitelist of allowed values. We should add a link to a reference in the future documentation. We might be able to hide this option from the public API for now."
     ->param('filters', [], new ArrayList(), 'Array of filters.')
     ->inject('response')
-    ->inject('dbForInternal')
+    ->inject('dbForExternal')
     ->inject('audits')
-    ->action(function ($collectionId, $id, $type, $size, $required, $signed, $array, $filters, $response, $dbForInternal, $audits) {
+    ->action(function ($collectionId, $id, $type, $size, $required, $signed, $array, $filters, $response, $dbForExternal, $audits) {
         /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Database $dbForInternal*/
+        /** @var Utopia\Database\Database $dbForExternal*/
         /** @var Appwrite\Event\Event $audits */
 
```

Comment:
We should first fetch the collection by ID, check it's actually exist and continue or throw 404 exception.